iothub/device/src/Authentication/Security/SharedAccessSignature.cs

```diff
@@ -2,152 +2,36 @@
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
 using System;
-using System.Collections.Generic;
-using System.Globalization;
 using System.Net;
-using System.Security.Cryptography;
-using System.Text;
 
 namespace Microsoft.Azure.Devices.Client
 {
-    internal sealed class SharedAccessSignature : ISharedAccessSignatureCredential
+    internal sealed class SharedAccessSignature
```

Comment:
This class now represents only the parsed out shared access signature. The parsing logic has been moved to its own class. I'll check in a subsequent PR if this parsing is really required.